.github/workflows/revdepcheck.yaml

```diff
@@ -0,0 +1,94 @@
+---
+name: revdepcheck ⏪
+
+on:
+  workflow_dispatch:
```

Comment:
copying @Melkiades comment that I agree with

> I think it is very helpful to understand how to minimize breaking changes, hence to communicate them. I would keep it running every couple of days + manual runs if needed

Reply:
The easiest way would be to add revdepcheck to the `scheduled.yaml` workflow, like [this](https://github.com/insightsengineering/tern/blob/dc5f4a86d7aa6bf76561d83df224eaa40027196e/.github/workflows/scheduled.yaml#L34-L36).

But then if someone wants to run revdepcheck on demand, they'll also trigger other reusable workflows from this file. If revdepcheck is part of this workflow, it will also run on the same schedule defined in this file.

Reply:
I personally like this: https://github.com/insightsengineering/rtables/actions/workflows/scheduled.yaml
I do not if it is what you mean

Reply:
What Franek is saying is to put revdepcheck _next to_ verdepcheck and make them always run together - triggered either manually or on schedule.
While conceptually those are quite similar and we want both to be run on a schedule - there are some differences in the scope:
- verdepcheck is testing downstream deps specification in DESCRIPTION file whereas revdepcheck is testing upstream deps for potential breaking changes
- if revdepcheck fails and I am fixing this - I don't really need to repeat verdepcheck (and vice versa)

Therefore I think I it's not good to put them together. I would prefer to introduce schedule separately.

(This makes "Scheduled" action label a little bit misleading but I think it's a separate issue)

Reply:
Of course we can move revdepcheck to a separate workflow, like [here](https://github.com/insightsengineering/tern/actions/runs/8706549600).
This way just one particular workflow can be called on demand.
The naming can also be adjusted.

One thing to keep in mind is that those kind of changes (workflow triggers, and which calls of reusable workflows are grouped in one workflow file) have to be defined at the package repository level. So it's worth making sure we agree on triggers/schedules/naming etc. to be used across repositories, because if we would like to change that in the future, it will require a propagation.